src/components/MemberActivity/Modal/CustomCell/customTableCell.tsx

```
@@ -43,15 +47,18 @@ export const CustomTableCell: React.FC<{ eventId: string }> = ({ eventId }) => {
   if (loading)
     return (
       <TableRow data-testid="loading-state">
-        <TableCell colSpan={4}>
-          <CircularProgress />
+        <TableCell colSpan={colSpan} align="center" aria-busy="true">
+          <span role="status" aria-live="polite">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)